MAINTAINERS.md

```diff
@@ -0,0 +1,50 @@
+# How to release a new version
+
+## When to release a new version
+
+New version should be released when we are ready to make changes generally available.
+
+New version should not be released if we want to test our latest changes or to make them available to a limited number of users. This can be achieved without releasing a new version. Go modules allow to point to latest master or specific Git commit. However, commits from master are not suitable for use in production unless they are a release tag. Do not use non-release commits in downstream projects in production.
+
+Consider releasing a new Release Candidate version to make changes available to a larger group of users, if we are not ready to make them available to everyone yet.
+
+Under no circumstances may releases be done during the weekend or the wee hours of the night.
```

Comment:
Who's night?

Reply:
The one doing the release. The rationale is that this is not a task you should do when you are supposed to sleep.